nucypher/blockchain/eth/actors.py

```diff
@@ -1064,6 +1104,17 @@ def _deposit(self, amount: int, lock_periods: int) -> TxReceipt:
                                                         call_data=Web3.toBytes(lock_periods))
         return receipt
 
+    def _lock_and_create(self, amount: int, lock_periods: int) -> TxReceipt:
+        """Public facing method for token locking without depositing."""
+        # TODO #1497 #1358
```

Comment:
:thinking:  Do we know if this is still an outstanding issue?

Reply:
I think after we will see first pooling contracts and demand for CLI for them then will be better understanding what to do we those issues